src/exercises/day-3/simple-gui.rs

```diff
@@ -155,6 +159,17 @@ fn main() {
         "Click me!",
         Box::new(|| println!("You clicked the button!")),
     )));
+    let mut inner_window = Box::new(Window::new("Sub-window"));
```

Comment:
This now shows up in the exercise text, right? The exercise shows the expected output and this is now out of sync. I would like to remove the code from here to keep things simple in the exercise.

Instead,  we could move the code to a unit test — it would be nice to have tests for this code in general. Would you be up for writing a few simple unit tests below?